.travis.yml

```diff
@@ -45,7 +45,13 @@ jobs:
       script: make build
       os: osx
     - stage: test
-      script: true
+      script:
+        - go test -v $(go list ./... | grep -vE "vendor|cli|utils")
+        # Allow the cli tests to fail, as some of them rely on external tooling,
+        # such as debugfs.
+        # The utils tests are dropped, as it assumes the existence of /bin/true,
+        # which was not found on osx.
+        - go test -v github.com/mendersoftware/mender-artifact/cli/mender-artifact || true
```

Comment:
So I guess the question is:
Do you want it handled more precisely? ie, running tests individually,
or rather excluding tests that rely on external binaries manually?

Reply:
Yes! That is my point in the other comment. Sorry I didn't read this before. IMO, a "do whatever || true" cannot be a good idea.